toast.nim

```diff
@@ -126,11 +127,15 @@ proc main(
     debug: debug,
     defines: defines,
     includeDirs: includeDirs,
-    symOverride: symOverride
+    symOverride: symOverride,
+    pluginFile: pluginFile
```

Comment:
generally applicable tip:
always end with `,`, that way the diff only shows 1 line instead of 2 everytime u add a field, eg:
```
symOverride: symOverride,
pluginFile: pluginFile,
```